README.rst

```diff
@@ -40,15 +40,28 @@ for the full list of configuration options and more details on each.
 Installation
 ------------
 
-With `pathogen.vim`_:
+With `Pathogen.vim`_:
 
 .. code:: bash
 
-  $ cd ~/.vim/bundle
-  $ git clone https://github.com/mtth/scratch.vim
+  git clone https://github.com/mtth/scratch.vim ~/.vim/bundle/scratch
+
+With `Vundle.vim`_:
```

Comment:
Can you add a small comment mentioning that the following snippets should go in the `.vimrc` directly?